taichi/ir/ir.h

```diff
@@ -805,7 +806,7 @@ class Block : public IRNode {
  public:
   Block *parent;
   std::vector<std::unique_ptr<Stmt>> statements, trash_bin;
-  std::map<Identifier, Stmt *> local_var_alloca; // Only used in frontend
+  std::map<Identifier, Stmt *> local_var_alloca;  // Only used in frontend
```

Comment:
I guess this is changed due to the formatter, but since I see it now, how about:

```suggestion
  std::unordered_map<Identifier, Stmt *> local_var_alloca;  // Only used in frontend
```

Reply:
Good idea! But we want to focus in this PR and not getting unwanted bugs into here.